### PR TITLE
Add instance age metric

### PR DESCRIPTION
**Pull request by vmercierfr**
# Objective

Add instance age metric

# Why

Some workloads require raising alerts based on instance age.

At Qonto, we raise alerts when some instances (based on name patterns) run too long to prevent uncontrolled AWS costs.

# How

- Add `rds_instance_age_seconds`
- Document how to add a new metric

# Release plan

- [ ] Merge this PR

### CODE REVIEW

**Comment by vmercierfr** [comment]
Reviewer's note: The `rds_instance_uptime` metric in `rds-management` was a little misleading, as it appears to be the time since the RDS instance was last started, when it is the creation date. `rds_instance_age_seconds` sounds more accurate.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ It collect key metrics about:
 | rds_exporter_errors_total | | Total number of errors encountered by the exporter |
 | rds_free_storage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Free storage on the instance |
 | rds_freeable_memory_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Amount of available random access memory. For MariaDB, MySQL, Oracle, and PostgreSQL DB instances, this metric reports the value of the MemAvailable field of /proc/meminfo |
+| rds_instance_age_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | Instance age |
```
Comment (qfritz) [inline README.md:51]:
```suggestion
| rds_instance_age_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | Time since instance creation |
```
or something to avoid confusing it with uptime (which I agree, can be quite different).

**Review by qfritz — APPROVED** [review]
LGTM

Maybe if we have the need for it, future releases can include an `rds_instance_uptime` which will completely disambiguate `rds_instance_age`.

File: README.md [hunk 2]
```diff
@@ -64,10 +65,10 @@ It collect key metrics about:
...
 | rds_read_iops_average | `aws_account_id`, `aws_region`, `dbidentifier` | Average number of disk read I/O operations per second |
 | rds_read_throughput_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Average number of bytes read from disk per second |
-| rds_transaction_logs_disk_usage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Disk space used by transaction logs (only on PostgreSQL) |
 | rds_replica_lag_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | For read replica configurations, the amount of time a read replica DB instance lags behind the source DB instance. Applies to MariaDB, Microsoft SQL Server, MySQL, Oracle, and PostgreSQL read replicas |
 | rds_replication_slot_disk_usage_average | `aws_account_id`, `aws_region`, `dbidentifier` | Disk space used by replication slot files. Applies to PostgreSQL |
 | rds_swap_usage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Amount of swap space used on the DB instance. This metric is not available for SQL Server |
+| rds_transaction_logs_disk_usage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Disk space used by transaction logs (only on PostgreSQL) |
```
Comment (dcupif) [inline README.md:71]: Seems unrelated to this MR, was this forgotten in a previous MR?

**Review by dcupif — APPROVED** [review]
lgtm 🚀

File: README.md (hunk 1 above)
Comment (vmercierfr) [reply README.md:51]: I applied the change (and reflected it in metric help in `exporter.go`).

Note: So far, I didn't see metrics in RDS API or Cloudwatch API to collect the uptime metric. In our context, we have it using PostgreSQL exporter.

File: README.md (hunk 2 above)
Comment (vmercierfr) [reply README.md:71]: It's just alphabetical order (lien 57 and 71), I changed the commit history to reflect this change.
